dcos/dcos-ui
# DCOS-51269: COPS-4710 affecting : Package-specific icons not showing for some services installed from Universe

Add icons for services that are installed from
Universe, but are not classified as Frameworks.

Closes https://jira.mesosphere.com/browse/DCOS-51269

## Testing
1. Start some universe services that are not classified as frameworks, for example "spark-history" or "kibana".
2. Check that they have their own icons and not the default ones (as icons don't work in production, you can inspect the element to see the link and verify that they are showing a missing icon and not the default one).
3. Check the same thing for services inside folders.
4. Verify that no universe or not-universe service breaks the UI.

## Trade-offs
This is work in progress. I have to update/add tests, but it's otherwise reviewable.

## Dependencies
None.

## Screenshots
### Before
![Screenshot from 2019-04-12 15-07-49](https://user-images.githubusercontent.com/40791275/56036171-d41f1480-5d34-11e9-9030-b90a1226c094.png)
![Screenshot from 2019-04-12 15-08-13](https://user-images.githubusercontent.com/40791275/56036172-d6816e80-5d34-11e9-9e16-5185f1a639a1.png)

### After
![Screenshot from 2019-04-12 14-53-52](https://user-images.githubusercontent.com/40791275/56036045-7e4a6c80-5d34-11e9-9e63-3c6cc591b91e.png)
![Screenshot from 2019-04-12 14-54-14](https://user-images.githubusercontent.com/40791275/56036054-86a2a780-5d34-11e9-8660-c699796ff99f.png)
